server/internal/cache/cache.go

```diff
@@ -100,7 +101,10 @@ func (c *Store) Sync(ctx context.Context, interval time.Duration) error {
 func (c *Store) updateCache(ctx context.Context) error {
 	resp, err := c.userInfoLister.ListAPIKeys(ctx, &uv1.ListAPIKeysRequest{})
 	if err != nil {
-		return err
+		// Gracefully ignore the error.
+		// TODO(kenji): Make the pod unready.
+		log.Printf("Failed to list API keys: %s. Ignoring.", err)
+		return nil
```

Comment:
How about ignoring updateCache errors in sync loop to handle both apikeys and orgusers?

Reply:
sounds good!

Reply:
done